yjit/src/disasm.rs

```diff
@@ -1,9 +1,12 @@
 use crate::core::*;
 use crate::cruby::*;
 use crate::yjit::yjit_enabled_p;
+#[cfg(feature = "disasm")]
 use crate::asm::CodeBlock;
+#[cfg(feature = "disasm")]
 use crate::options::DumpDisasm;
 
+#[cfg(feature = "disasm")]
 use std::fmt::Write;
```

Comment:
Guarding a bunch of individual imports like that seems to me like it could trip people up, wouldn't really want to write code like that?

Reply:
Why? This code is only used when feature=disasm. Why would our code pretend that isn't the case We have these cfg flags all over the place, so I'm really sure I see the difference here. 

I'm down with the alternative of moving them to codeblocks where they are actually used.